test/functional/workspace/test_workspace.py

```diff
@@ -47,3 +47,27 @@ def test_metabuild():
     assert "Adding project app1" in c.out
     c.run_command("cmake --build --preset conan-release")
     # it doesn't fail
+
+
+def test_relative_paths():
+    # This is using the meta-project
+    c = TestClient()
+    c.run("new cmake_lib -d name=liba --output=liba")
+    c.run("new cmake_exe -d name=app1 -d requires=liba/0.1 --output=app1")
+    c.run("new cmake_lib -d name=libb --output=other/libb")
+    c.run("new cmake_exe -d name=app2 -d requires=libb/0.1 --output=other/app2")
+    c.run("workspace init mywks")
+    c.run("workspace init otherwks")
+    # cd mywks
+    with chdir(os.path.join(c.current_folder, "mywks")):
+        breakpoint()
+        c.run("workspace add ../liba")
+        c.run("workspace add ../app1 --product")
+        c.run("workspace build")
+        assert "[100%] Built target app1" in c.out
+    # cd otherwks
+    with chdir(os.path.join(c.current_folder, "otherwks")):
+        c.run("workspace add ../other/libb")
+        c.run("workspace add ../other/app2 --product")
+        c.run("workspace build")
+        assert "[100%] Built target app2" in c.out
```

Comment:
Executing the test manually works like a charm, but over here, it fails. Looking into it.